scripts/foosbot.coffee

```diff
@@ -694,6 +695,60 @@ returnFromCleanseRespond = (res) ->
     res.reply "Welcome back! Now go kick some ass"
 
 
+getTeamStats = (playerOneName, playerTwoName) ->
+    stats = {
+        "wins": 0,
+        "losses": 0,
+        "goalsFor": 0,
+        "goalsAgainst": 0
+    }
+
+    for finishedGame in finishedGames
+        teamOne = [finishedGame["team1"]["player1"], finishedGame["team1"]["player2"]]
+        teamTwo = [finishedGame["team2"]["player1"], finishedGame["team2"]["player2"]]
+
+        if playerOneName in teamOne && playerTwoName in teamOne
+            myTeam = finishedGame["team1"]
+            otherTeam = finishedGame["team2"]
+        else if playerOneName in teamTwo && playerTwoName in teamTwo
+            myTeam = finishedGame["team2"]
+            otherTeam = finishedGame["team1"]
+        else
+            continue
+
+        if myTeam["score"] > otherTeam["score"]
+            stats["wins"] += 1
+        else
+            stats["losses"] += 1
```

Comment:
Maybe do nothing if the scores are equal. They shouldn't be if people enter scores correctly, but the code should still handle it gracefully, since nothing really stops anyone from finishing a game with equal scores.